### DISCUSSION TITLE
Is this project now abandoned?

### DISCUSSION THREAD

**Discussion by GodsBadAssBlade** [ChristophHaag/SteamVR-OpenHMD#84] [General]
It'd be a shame for something like this to go nowhere, It'd be a godsend for a large amout of people who not only use linux but own vr's if it really gained some steam and community support. Maybe a shout to steam could result in help from them since theres at least proof of concept?

**Comment by ChristophHaag** [comment]
The only thing that's really happening with OpenHMD is thaytan's fork for the Rift CV1, otherwise it's pretty much abandoned, so I don't feel like putting in a lot of effort.

Instead, Monado's SteamVR plugin is essentially an improved version of this plugin.

https://monado.freedesktop.org/
https://monado.freedesktop.org/steamvr.html
